notebooks/238-attention-center/238-attention-center.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>Eye tracking is commonly used in visual neuroscience and cognitive science to answer related questions such as visual attention and decision making. Computational models that predict where to look have direct applications to a variety of computer vision tasks. The attention center model takes an RGB image as input and return a 2D point as output. This 2D point is the predicted center of human attention on the image i.e. the most salient part of images, on which people pay attention fist to. This allows find the most visually salient regions and handle it as early as possible. For example it could be used for the latest generatipon image format(such as JPEG XL), which supports encoding the parts that you pay attention to fist. It can help to improve user experience, image will appear to load faster.</p><p> </p><p>(also it will be nice if you can provide link on info about JPEG XL)</p><p> </p><p>The attention center model has been trained with images from the COCO dataset annotated with saliency maps from the <a href="http://salicon.net/" target="_blank">salicon dataset</a>.</p><p> </p><p><a href="https://opensource.googleblog.com/2022/12/open-sourcing-attention-center-model.html" target="_blank"><s>[Open sourcing the attention center model]</s></a><s> </s> Source: [google AI blogpost](https://opensource.googleblog.com/2022/12/open-sourcing-attention-center-model.html)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>also I think you can past image https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEjxLCDJHzJNjB_von-vFlq8TJJFA41aB85T-QE3ZNxW8kshAf3HOEyIEJ4uggXjbJmZhsdj7j6i6mvvmXtyaxXJPm3JHuKILNRTPfX9KvICbFBRD8KNuDVmLABzYuhQci3BT2BqV-wM54IxaoAV1YDBbnpJC92UZfEBGvakLusiqND2AaPpWPr2gJV1/s1600/image4.png</p>

Reply:
Do I need to remove paragraph about architecture?

Reply:
no, just refomat quatas source and add image

Reply:
Also I find that on the latest dev openvino performance is not looks good:
IR model in OpenVINO Runtime/CPU: 0.7983 seconds per image, FPS: 1.25
TensorFlow API: 0.4376 seconds per image, FPS: 2.29

Reply:
> no, just refomat quatas source and add image

ok, thank you, done

Reply:
@eaidova inference with TensorFlow removed